db/system_keyspace.cc

```diff
@@ -264,6 +264,8 @@ schema_ptr system_keyspace::topology() {
             .with_column("new_cdc_generation_data_uuid", timeuuid_type, column_kind::static_column)
             .with_column("new_keyspace_rf_change_ks_name", utf8_type, column_kind::static_column)
             .with_column("new_keyspace_rf_change_data", map_type_impl::get_instance(utf8_type, utf8_type, false), column_kind::static_column)
+            .with_column("truncate_table_id", uuid_type, column_kind::static_column)
+            .with_column("truncate_table_timeout_ms", long_type, column_kind::static_column)
```

Comment:
The timeout shouldn't be here. The truncate has to complete, if someone asked for it. The timeout is limited to the CQL coordinator waiting for it to complete.

Reply:
This was implemented in v7. Closing conversation.